bot.py

```diff
@@ -31,4 +31,41 @@ async def create_repo(ctx, repoName): #!git create_repo repo1
 	repo = user.create_repo(repoName)
 	await ctx.send("repository "+repoName+ " created!")
 
+
+# display open and closed issues
+
+@bot.command()
+async def issues(ctx, repoName, state): # !git issues MLH-Fellowship/github-discord-bot open
+	repo = g.get_repo(repoName)
+	issues = repo.get_issues(state=state)
+	if(issues.totalCount == 0):
+		await ctx.send("There are no issues that match your query")
+	else: 
+		for i in issues:
+			await ctx.send('Issue Title: ' + i.title + '\nIssue Number: ' + str(i.number) +'\nIssue Link: https://github.com/' + repoName + '/issues/' + str(i.number))
+
+
+# display open pull requests
+
+@bot.command()
+async def pull_requests(ctx, repoName, state): # !git pull_requests MLH-Fellowship/github-discord-bot open
+	repo = g.get_repo(repoName)
+	pulls = repo.get_pulls(state=state, sort='created')
+	if(pulls.totalCount == 0):
+		await ctx.send("There are no pull requests that match your query")
+	else:
+		for pr in pulls:
+			await ctx.send('Pull Request Title: ' + pr.title + '\nPull Request Number: ' + str(pr.number) +'\nPull Request Link: https://github.com/' + repoName + '/pull/' + str(pr.number))
```

Comment:
:bike: Kind of a nit: The code here seems repetitive, maybe there could be a helper function that gets *things* (like PRs and issues) to which you'd pass in a function that it would then execute. I mean, if you wanna go really fancy it could even be a [decorator](https://realpython.com/primer-on-python-decorators/).